docs/content/configuration/ingress-resources/basic-configuration.md

```diff
@@ -63,17 +69,21 @@ Starting from Kubernetes 1.18, you can use the following new features:
     - path: /tea/green
       pathType: Exact
       backend:
-        serviceName: tea-svc
-        servicePort: 80
+          service:
+            name: tea-svc
+            port:
+              number: 80
     - path: /coffee
       pathType: ImplementationSpecific # default
```

Comment:
it is mandatory, not default in v1, right? shall we remove the default comment?

I think we need to rewrite this doc a bit and merge `New Features Available in Kubernetes 1.18 and Above` into the rest of the doc, because with NIC 1.13, there will be only v1. I will create a PR for that